ui/component/or-dashboard-builder/src/panels/thresholds-panel.ts

```diff
@@ -173,13 +173,14 @@ export class ThresholdsPanel extends LitElement {
                 ` : null}
 
                 <!-- Thresholds by boolean -->
-                ${(this.valueType === 'boolean') ? html`
+                ${(this.valueType === 'boolean' && this.boolColors) ? html`
                     <div class="threshold-list-item">
                         <div class="threshold-list-item-colour">
                             <or-mwc-input type="${InputType.COLOUR}" value="${this.boolColors.true}"
                                           @or-mwc-input-changed="${(event: OrInputChangedEvent) => {
-                                              this.boolColors.true = event.detail.value;
-                                              this.requestUpdate('boolColors');
+                                          console.log("In threshold bool colors true", this.boolColors);    
```

Comment:
Maybe remove the console logs? @mihaelaaleks

Reply:
Good catch, ty